sycl/doc/design/HostPipes.md

```diff
@@ -0,0 +1,193 @@
+# Implementation design for "Host Pipes"
+
+This document describes the implementation design for the host pipes section
+of the DPC++ extension [SYCL_INTEL_data_flow_pipes][1]. Pipes are a FIFO construct 
+that provide links between elements of a design that are accessed through read 
+and write application programming interfaces (APIs), without the notion of a
+memory address/pointer to elements within the FIFO. A host pipe is a pipe that 
+links a device kernel with a host program.
+
+[1]: https://github.com/intel-sandbox/ip-authoring-specs/blob/main/Pipe/Spec/data_flow_pipes.asciidoc
```

Comment:
```suggestion
[1]: <../extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc>
```
@gmlueck - Do you know if it is intentional that the sycl_ext_intel_dataflow_pipes extension is under supported extensions?

Reply:
Yes, this is intentional.  The dataflow pipes extension was created before we reorganized the extension documents.  My understanding is that there is production FPGA application code using that extension, so we decided to make it "supported".